examples/java/src/main/java/com/xebia/functional/xef/java/auto/BreakingNews.java

```diff
@@ -0,0 +1,31 @@
+package com.xebia.functional.xef.java.auto;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class BreakingNews {
+
+    public String summary;
+
+    static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd/M/yyyy");
+    static LocalDateTime now = LocalDateTime.now();
+
+
+    private static CompletableFuture<Void> writeParagraph(AIScope scope) {
+        var currentDate = dtf.format(now);
+
+        scope.prompt("write a paragraph of about 300 words about: " + currentDate + " Covid News", BreakingNews.class)
+              .thenAccept(breakingNews -> System.out.println(currentDate + " Covid news summary:\n" + breakingNews.summary));
+
+        return writeParagraph(scope);
```

Comment:
writeParagraph is calling itself, so it recurses and runs out of stack space

Reply:
Done, but now is returning java.lang.OutOfMemoryError: Java heap space in Java example and Kotlin example

Reply:
I am not seeing the error anymore